### ISSUE TITLE
Release v0.2.0?

### ISSUE THREAD

**Issue by zuphilip** [UB-Mannheim/ocr-fileformat#29]
I think we should create a new release. I started to draft one in GitHub, see https://github.com/UB-Mannheim/ocr-fileformat/releases . However, I am not sure, what has to been done with the release option in the [Makefile](https://github.com/UB-Mannheim/ocr-fileformat/blob/master/Makefile). Is it enough to increase the version counter? Do you agree that we are now at `v0.2.0`?

**Comment by stweil** [comment]
@zuphilip, before releasing we should review our license. Is MIT compatible with CC-BY-SA?

**Comment by zuphilip** [comment]
I think it is okay, because everything here in the repo can be licensed under MIT and the XSD files are copied during the make process. But maybe we should clarify that more in the README?

**Comment by kba** [comment]
* [ ] Merge https://github.com/UB-Mannheim/ocr-fileformat/tree/to-text
* [ ] Merge https://github.com/UB-Mannheim/ocr-fileformat/tree/gcv (kinda blocked by   dinosauria123/gcv2hocr#3)
* [ ] Revise and merge https://github.com/UB-Mannheim/ocr-fileformat/tree/better-testing (since that would be helpful indeed).

**Comment by zuphilip** [comment]
These are good ways to continue, but we can IMO also create a new release before finishing these new features. There are already quite some changes since `v0.1.0`, don't you think? See https://github.com/UB-Mannheim/ocr-fileformat/releases/tag/untagged-7d39472a88449d704f13

**Comment by kba** [comment]
Yes, good point, I forgot the whole "code cleanup and backend rewrite" part. We can release that now and maybe schedule the outstanding merges in another issue.

**Comment by zuphilip** [comment]
Okay, I moved the points from above to a new issue and just prepared a new release. Please look at the PR which should then close this issue here.

**Comment by kba** [comment]
@stweil reCC-BY-SA / MIT: Combining licenses is always confusing. We could try to get in touch with @filak to ask for licensing terms better suited for software. There's also the [ocr-xsl](https://github.com/kba/ocr-xsl) project that I started to write something from scratch but have no time for at the moment.

But I think we're being honest and clear in the README about software we bundle and what is covered by our license vs. how bundled software is licensed.

**Comment by filak** [comment]
OK, what do you suggest? I can use another license (public domain CC0?) and make a new commit. But if you just leave the licensing lines in the files then you shall be fine so far.

**Comment by kba** [comment]
Hi Filip, thanks for answering! The problem is not attribution but that a stylesheet is software not a creative work, so Creative Commons is not a good fit in general. CC0 is possible for software as well but IMHO you can't go wrong with MIT, it's a permissive widely-used license, attribution because the license notice must be reproduced and it's legally uncomplicated to extend/build on top of MIT-licensed software

**Comment by filak** [comment]
How about the Apache License 2.0 ? Would it be ok with you?

**Comment by kba** [comment]
@zuphilip 
> However, I am not sure, what has to been done with the release option in the Makefile.
This creates archives `ocr-fileformat_VERSION.tar.gz` and ``ocr-fileformat_VERSION.zip` that contain all the tools. I've added those to the release page.

**Comment by kba** [comment]
@filak In principle I have nothing against Apache License, it's broadly the same as MIT but with lots of additional safeguards regarding trademarks, patents, requiring users to state changes made etc. A bit overkill for a small project but still a good choice.

**Comment by filak** [comment]
Nevermind, I have just done it with the MIT: https://github.com/filak/hOCR-to-ALTO/commit/80ae1f9f2a0d1d5d54450b1f6592c5a82ee8561d
